src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -894,3 +907,108 @@ export function getGroupsDownloadData(
     }
     return lines.map(line => line.join('\t')).join('\n');
 }
+
+export function buildFilterMenu(
```

Comment:
this should probably be a functional component that gets tagged as observer

Reply:
Updated